.github/workflows/tests.yaml

```diff
@@ -203,6 +203,22 @@ jobs:
         if: steps.filter.outputs.has_code == 'true'
         run: df -h
 
+      - name: Verify MODULE.bazel.lock
+        if: steps.filter.outputs.has_code == 'true'
+        run: |
+          exit_code=0
+          bazelisk mod deps --lockfile_mode=error || exit_code=$?
+          if (( $exit_code != 0 )); then
+            bazelisk mod deps --lockfile_mode=update
+            echo "MODULE.bazel.lock is out of date! Use below file for update."
+            echo "Platforms may require merging output, for example by applying"
+            echo "an update, re-running triggers, and applying the next update."
+            echo "============================================================"
+            cat MODULE.bazel.lock
+            echo "============================================================"
```

Comment:
```suggestion
            echo "MODULE.bazel.lock is out of date! Use below patch for update."
            echo "You may also need to apply patches for other platforms."
            echo "============================================================"
            git diff MODULE.bazel.lock
            echo "============================================================"
```
... maybe?

Reply:
I'd started with something similar to that, but I was trying to clarify what it meant to apply patches for other platforms. It's really just copy-pasting the whole file, overwriting the old one.